aspire-orchestrator/Aspire.AppHost/Program.cs

```diff
@@ -6,34 +6,111 @@
+        break;
 }
 
 builder.Build().Run();
+
+IResourceBuilder<ExecutableResource> AddWorkbenchBackend()
+{
+    return builder.AddWorkbenchService(
+        name: "workbenchservice",
+        projectDirectory: Path.Combine("..", "..", "workbench-service"),
+        clientId: clientId);
+}
+
+IResourceBuilder<NodeAppResource> AddWorkbenchFrontend(EndpointReference workbenchServiceEndpoint)
+{
+    return builder.AddViteApp(
+            name: "workbenchapp",
+            workingDirectory: Path.Combine("..", "..", "workbench-app"),
+            packageManager: "pnpm")
+        .WithPnpmPackageInstallation()
+        .WithEnvironment(name: "VITE_SEMANTIC_WORKBENCH_SERVICE_URL", workbenchServiceEndpoint)
+        .WaitFor(workbenchBackend)
+        .PublishAsDockerFile([
+            new DockerBuildArg("VITE_SEMANTIC_WORKBENCH_CLIENT_ID", clientId.Resource.Value),
+            new DockerBuildArg("VITE_SEMANTIC_WORKBENCH_AUTHORITY", authority.Resource.Value),
+        ]);
+}
+
+// .NET Agent Example 1
+IResourceBuilder<ProjectResource> AddDotnetAgent1(EndpointReference workbenchServiceEndpoint)
+{
+    var agent = builder
+        .AddProject<Projects.dotnet_01_echo_bot>(name: "echo-bot-dotnet")
+        .WithHttpEndpoint()
+        .WaitFor(workbenchBackend)
+        .WithEnvironment(name: "Workbench__WorkbenchEndpoint", workbenchServiceEndpoint);
+
+    return agent.WithEnvironment("Workbench__ConnectorHost", $"{agent.GetEndpoint("http")}");
+}
+
+// .NET Agent Example 2
+IResourceBuilder<ProjectResource> AddDotnetAgent2(EndpointReference workbenchServiceEndpoint)
+{
+    var agent = builder
+        .AddProject<Projects.dotnet_02_message_types_demo>(name: "sw-tutorial-bot-dotnet")
+        .WithHttpEndpoint()
+        .WaitFor(workbenchBackend)
+        .WithEnvironment(name: "Workbench__WorkbenchEndpoint", workbenchServiceEndpoint);
+
+    return agent.WithEnvironment("Workbench__ConnectorHost", $"{agent.GetEndpoint("http")}");
+}
+
+// .NET Agent Example 3
+IResourceBuilder<ProjectResource> AddDotnetAgent3(EndpointReference workbenchServiceEndpoint)
+{
+    var agent = builder
+        .AddProject<Projects.dotnet_03_simple_chatbot>(name: "simple-chatbot-dotnet")
+        .WithHttpEndpoint()
+        .WaitFor(workbenchBackend)
+        .WithEnvironment(name: "Workbench__WorkbenchEndpoint", workbenchServiceEndpoint);
+
+    return agent.WithEnvironment("Workbench__ConnectorHost", $"{agent.GetEndpoint("http")}");
+}
+
+// Python Agent example: Skill Assistant
+IResourceBuilder<ExecutableResource> AddPythonSkillAssistant(EndpointReference workbenchServiceEndpoint)
+{
+    return builder
+        .AddAssistantApp(
+            name: "skill-assistant",
+            projectDirectory: Path.Combine("..", "..", "assistants", "skill-assistant"),
+            assistantModuleName: "skill-assistant")
+        .WithEnvironment(name: "assistant__workbench_service_url", workbenchServiceEndpoint);
+}
+
+// Python Agent example: Multi-model chatbot
```

Comment:
@tommasodotNET I think we need some work in the UV extension to allow loading multiple Python agents. There seems to be some problem with `assistantModuleName` and port `3001` hard coded in `AddAssistantApp`.

Reply:
Current implementation should already allow that:
```csharp
    public static IResourceBuilder<ExecutableResource> AddAssistantApp(
        this IDistributedApplicationBuilder builder,
        string name,
        string projectDirectory,
        string assistantModuleName)
    {
        ArgumentNullException.ThrowIfNull(builder);

        var assistant = builder.AddUvApp(name, projectDirectory, "start-assistant")
            .PublishAsDockerImage(dockerContext: Path.Combine("..", ".."),
                dockerFilePath: Path.Combine("tools", "docker", "Dockerfile.assistant"),
                configure: new(configure => configure
                    .WithBuildArg("package", assistantModuleName)
                    .WithBuildArg("app", $"assistant.{assistantModuleName.Replace('-', '_')}:app")
                ));
        if (builder.ExecutionContext.IsPublishMode)
        {
            assistant.WithHttpEndpoint(port: 3001, env: "ASSISTANT__PORT");
        }
        else
        {
            assistant.WithHttpEndpoint(env: "ASSISTANT__PORT");
        }
...
```
Currently, the 3001 is hardcoded only in publish mode, when the containers ports won't get into conflict. Locally, a random port should be assigned to each assistant by Aspire using the env var ASSISTANT__PORT.

Reply:
@tommasodotNET Could you try uncommenting the block below? I can't find the exact param values to pass to start multi-model chatbot

Reply:
found the problem, the path was incorrect